### ISSUE TITLE
Fallback value for unset variable is removing spaces

### ISSUE THREAD

**Issue by franciscocpg** [serverless/serverless#5398]
# This is a Bug Report

## Description

For bug reports:
* What went wrong?

Given the following variable fallback:
```yaml
events:
      - schedule: ${env:SCHEDULE, 'rate(2 hours)'}
```

If env `SCHEDULE` is not set I got this error when I run `sls deploy`:
```
An error occurred: ScheduleEventsRuleSchedule1 - Parameter ScheduleExpression is not valid. (Service: AmazonCloudWatchEvents;Status Code: 400; Error Code: ValidationException; Request ID: 996bca35-d0bb-11e8-92f7-2963d8953995).
```

And when I take a look at `serverless-state.json` I see this:
```json
"ScheduleExpression": "rate(2hours)",
```

So it looks like the fallback is removing the spaces. Is this expected?

* What did you expect should have happened?

I expect `sls deploy` to run sucessfully falling back to `rate(2 hours)`.

* What was the config you used?

See above

* What stacktrace or error message from your provider did you see?

See above
## Additional Data

* ***Serverless Framework Version you're using***:

1.32.0

* ***Operating System***:

Linux

* ***Stack Trace***:

n/a

* ***Provider Error messages***:

n/a

**Comment by franciscocpg** [comment]
The culprit:
https://github.com/serverless/serverless/blob/33b7784bad10347767d447dd1c650e0d90931d49/lib/classes/Variables.js#L267-L271

If I just remove `.replace(/\s/g, '')` at line 71 and run tests nothing is broken but I'm afraid that this could have some side effect that tests are not covering?

Can I open a PR removing it?

**Comment by exoego** [comment]
I think this is already fixed in #5571 and #5594, and released in v1.35.1 and later.

**Comment by exoego** [comment]
This issue is still reproducible in `v1.36.3` 

Given 

```
service: aws-nodejs

provider:
  name: aws
  runtime: nodejs8.10

functions:
  hello:
    handler: handler.hello
    events:
      - schedule: ${env:SCHEDULE, 'rate(2 hours)'}
```

then 

```
$ sls print

service: aws-nodejs
provider:
  name: aws
  runtime: nodejs8.10
functions:
  hello:
    handler: handler.hello
    events:
      - schedule: rate(2hours)
```
